runtime/darwinia/src/pallets/tx_pause.rs

```diff
@@ -19,15 +19,27 @@
 // darwinia
 use crate::*;
 
-/// Calls that cannot be paused by the tx-pause pallet.
 pub struct TxPauseWhitelistedCalls;
 impl frame_support::traits::Contains<pallet_tx_pause::RuntimeCallNameOf<Runtime>>
 	for TxPauseWhitelistedCalls
 {
 	fn contains(full_name: &pallet_tx_pause::RuntimeCallNameOf<Runtime>) -> bool {
-		matches!(
-			(full_name.0.as_slice(), full_name.1.as_slice()),
-			(b"System", b"remark_with_event")
+		let pallet = full_name.0.as_slice();
+
+		// Pallets that can be paused by the tx-pause pallet.
+		!matches!(
+			pallet,
+			b"Balances"
+				| b"Assets" | b"Vesting"
+				| b"Deposit" | b"AccountMigration"
+				| b"DarwiniaStaking"
+				| b"Ethereum" | b"EVM"
+				| b"MessageTransact"
+				| b"BridgePolkadotGrandpa"
+				| b"BridgePolkadotParachain"
+				| b"BridgeDarwiniaMessages"
+				| b"BridgeDarwiniaDispatch"
+				| b"DarwiniaFeeMarket"
```

Comment:
Would you like to expand the current `impl_maintenance_tests` to ensure this works as intended.